packages/omg-js-rootchain/src/rootchain.js

```diff
@@ -179,8 +180,26 @@ class RootChain {
     )
     const address = await this.plasmaContract.methods.exitsQueues(hashed).call()
     const contract = getContract(this.web3, priorityQueueAbi.abi, address)
-    const exitQueue = await contract.methods.heapList().call()
-    return exitQueue.slice(1)
+    const rawExitQueue = await contract.methods.heapList().call()
+    const exitQueuePriorities = rawExitQueue.slice(1)
```

Comment:
How about a comment explaining why we call `rawExitQueue.slice(1)`?
IIRC it's because the first element in the returned `rawExitQueue` is always `0`

Reply:
done 👍